uiowa/uiowa
# Created a boolean field for card link button display

Resolves #4788 

### How to test:
1. `blt ds --site=default`
2. Head to https://default.uiowa.ddev.site/admin/config/sitenow/uiowa-pages
3. Check that the "Display card arrow button" is already on. Save.
4. Create a new card and see if "Show button" is defaulting to "Use site default." (The dropdown will only show if there's a link but no link text.)
5. Check that the arrow button is showing on the card.
6. Test the 2 conditions (show and hide) for the "Show button" dropdown on the card and see if they behave as expected.
7. Change the "Display card arrow button" to unchecked https://default.uiowa.ddev.site/admin/config/sitenow/uiowa-pages. Save.
8. Head back to the card page and refresh.
9. Verify that the button has now disappeared, following the site default option.
10. Test the 2 conditions (show and hide) for the "Show button" dropdown on the card and see if they behave as expected.
11. `blt ds --site=hawkeyemarchingband.uiowa.edu`
12. https://hawkeyemarchingband.uiowa.ddev.site/admin/config/sitenow/uiowa-pages You should not see the option to set the card button for the site as this is a V2 site.
